frontend/src/scenes/actions/ActionStep.js

```diff
@@ -271,7 +280,13 @@ export class ActionStep extends Component {
     }
     URLMatching = ({ step, isEditor }) => {
         return (
-            <div className="btn-group" style={{ margin: isEditor ? '4px 0 0 8px' : '0 0 0 8px' }}>
+            <div
+                className="btn-group"
+                style={{
+                    margin: isEditor ? '4px 0 0 8px' : '0 0 0 8px',
+                    display: this.isUrlChecked(step) ? 'block' : 'none',
```

Comment:
This would be better as a render condition (like so: `this.isUrlChecked(step) & <div...`) than dynamic `display`.

Reply:
done in https://github.com/PostHog/posthog/pull/1680/commits/a4be717894223ae2ed6b0ae424f0fb4208b913c5